docs/notebooks/basic-scrna-tutorial.ipynb

```diff
@@ -0,0 +1,1193 @@
+{
```

Comment:
<p>Code below was switched from CPM to proportional fitting (PF). The text should be updated accordingly. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>I've updated, but language could be reworded.</p>